com.scanoss.sbom-workbench.yml

```diff
@@ -36,10 +34,10 @@ modules:
         url: https://github.com/scanoss/sbom-workbench/releases/download/v1.12.4/sbom-workbench-1.12.4-linux-x86_64.AppImage
         sha256: 53f9dcb75e837d76199e8d8f68ae2e262a61ab36a61a9aa7d8bd7899f77dc751
       - type: file
-        url: https://download.scanoss.com/sbom-workbench/flathub/com.scanoss.sbom-workbench.metainfo.xml
-        sha256: dbc3b0914f7587f3432d9b0dac2201e5a0defc3a18496ec87086d596f5035086
+        url: https://raw.githubusercontent.com/scanoss/sbom-workbench/main/.erb/flatpak/com.scanoss.sbom-workbench.metainfo.xml
```

Comment:
by linking to a branch you just ensure that the build WILL break next time there is a change.

Reply:
You're right, we will improve this. We will wait until our next release to merge the asset relocation.